client/my-sites/checkout/concierge-session-nudge/concierge-support-session/index.jsx

```diff
@@ -0,0 +1,221 @@
+								</span>
+							</li>
+							<li className="concierge-support-session__checklist-item">
+								<Gridicon
+									icon="checkmark"
+									className="concierge-support-session__checklist-item-icon"
+								/>
+								<span className="concierge-support-session__checklist-item-text">
+									{ translate(
+										'{{b}}Traffic:{{/b}} How to make your site search-engine friendly.',
+										{
+											components: { b: <b /> },
+											comment: "This is a benefit listed on a 'Purchase a call with us' page",
+										}
+									) }
+								</span>
+							</li>
+							<li className="concierge-support-session__checklist-item">
+								<Gridicon
+									icon="checkmark"
+									className="concierge-support-session__checklist-item-icon"
+								/>
+								<span className="concierge-support-session__checklist-item-text">
+									{ translate(
+										"{{b}}Site building tools:{{/b}} Learn how to create a site you're proud to share.",
+										{
+											components: { b: <b /> },
+											comment: "This is a benefit listed on a 'Purchase a call with us' page",
+										}
+									) }
+								</span>
+							</li>
+							<li className="concierge-support-session__checklist-item">
+								<Gridicon
+									icon="checkmark"
+									className="concierge-support-session__checklist-item-icon"
+								/>
+								<span className="concierge-support-session__checklist-item-text">
+									{ translate(
+										'{{b}}Content:{{/b}} What information to include and where it should go.',
+										{
+											components: { b: <b /> },
+											comment: "This is a benefit listed on a 'Purchase a call with us' page",
+										}
+									) }
+								</span>
+							</li>
+							<li className="concierge-support-session__checklist-item">
+								<Gridicon
+									icon="checkmark"
+									className="concierge-support-session__checklist-item-icon"
+								/>
+								<span className="concierge-support-session__checklist-item-text">
+									{ translate( "{{b}}And more:{{/b}} Tell our experts what you'd like to cover.", {
+										components: { b: <b /> },
+										comment: "This is a benefit listed on a 'Purchase a call with us' page",
+									} ) }
+								</span>
+							</li>
+						</ul>
+
+						<h4 className="concierge-support-session__sub-header">
+							{ translate(
+								'Reserve a 45-minute "Quick Start" appointment, and save %(saveAmount)s if you sign up today.',
+								{
+									args: {
+										saveAmount: formatCurrency( savings, currencyCode ),
+									},
+								}
+							) }
+						</h4>
+
+						<p>
+							<b>
+								{ translate(
+									'Book your call today for just {{del}}%(oldPrice)s{{/del}} %(price)s.',
+									{
+										components: { del: <del /> },
+										args: {
+											oldPrice: formatCurrency( fullCost, currencyCode ),
+											price: productDisplayCost,
+										},
+									}
+								) }
+							</b>{' '}
```

Comment:
I'm wondering what is the trailing `{' '}` for? 🤔

Reply:
This adds a space between the two sentences. Do you suggest doing it a different way?

<img width="681" alt="Screenshot 2019-07-01 at 4 57 22 PM" src="https://user-images.githubusercontent.com/1269602/60433121-a6ee4e00-9c21-11e9-91fa-c19fa1167bd3.png">

Reply:
Ah, got it. Sorry for the ignorance.